atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -243,7 +243,14 @@ def _get_logs_url(self):
             # We're importing this here in order to trap ImportError
             from koji import PathInfo
             pathinfo = PathInfo(topdir=self.koji_root)
-            url = urljoin(pathinfo.work(), pathinfo.taskrelpath(self.koji_task_id))
+            # Koji's work() outputs an invalid workdir, so we should fix it manually
+            # Replace first occurrence of double slash from the right
+            work_url = '/'.join(pathinfo.work().rsplit('//', 1))
```

Comment:
The double `//` is not what causes the broken link issue. But if you want to clean it, then in the constructor of this plugin just **remove** trailing slash. This would cause `PathInfo` to return the url without double forward slashes.
For instance:
```python
>>> koji.PathInfo(topdir='http://koji.example.com/kojiroot').work()
'http://koji.example.com/kojiroot/work'
>>> koji.PathInfo(topdir='http://koji.example.com/kojiroot/').work()
'http://koji.example.com/kojiroot//work'
```

The real problem is that when `urljoin` is used, it drops `'/work'` from the URL path. Regardless of whether or not the string returned by `PathInfo.work()` contains double forward slashes, `//`. 
Any variation of the following should fix the problem:
```python
url = '/'.join([pathinfo.work(), pathinfo.taskrelpath(self.koji_task_id)])
```